addons/udes_stock_package_move/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "UDES Stock Package Move",
```

Comment:
Can we call it UDES Stock Move ? At the moment we are moving packages but would be good to also move any quant.

Reply:
Same for the name of the module.

Reply:
Can we add a README, please?